src/player.cpp

```diff
@@ -5828,6 +5828,28 @@ static int bound_mod_to_vals(int val, int mod, int max, int min)
     return mod;
 }
 
+void player::print_health()
+{
+    if( !is_player() ) {
+        return;
+    }
+
+    if( healthy > 50 ) {
+        add_msg( m_good, _("You're very healthy, your wounds heal faster") );
```

Comment:
Shouldn't these lines have closing punctuation like a period or something at the end?